tests/mocks/pageContext.mock.json

```diff
@@ -0,0 +1,409 @@
+{
+  "site": {
```

Comment:
remove this file, use the smallest data set needed to run each test.
this will make it clear what are you testing